src/components/Buttons/RadioCheckbox.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component, PropTypes }        from 'react'
```

Comment:
`Checkbox` might make sense as it's on top level components folder, i.e. `src/components/Checkbox`